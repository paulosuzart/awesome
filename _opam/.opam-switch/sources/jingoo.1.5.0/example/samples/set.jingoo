{%- set hoge = "ok" -%}
{%- set foo, bar = ("foo", "bar") -%}
hoge: {{ hoge }}
foo: {{ foo }}
bar: {{ bar }}

{% set baz = 0 -%}
{%- set baz += 1 -%}
baz: {{ baz }}
{% set baz += 1 -%}
baz: {{ baz }}
{% set baz -= 1 -%}
baz: {{ baz }}
{% set baz *= 6 -%}
baz: {{ baz }}
{% set baz /= 2 -%}
baz: {{ baz }}
{% set baz %= 2 -%}
baz: {{ baz }}

{% set iterated = false -%}
{%- for item in [ 1, 2, 3 ] -%}
  {% set iterated = true %}
{%- endfor -%}
{%- if not iterated %}Did not iterate!{% endif %}
