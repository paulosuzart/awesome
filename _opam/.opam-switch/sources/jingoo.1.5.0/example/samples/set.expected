hoge: ok
foo: foo
bar: bar

baz: 1
baz: 2
baz: 1
baz: 6
baz: 3
baz: 1

Did not iterate!
